Objective-J/CommonJS/lib/objective-j.js

```diff
@@ -94,6 +94,7 @@ exports.run = function(args)
             print("  -I, --objj-include-paths       include a specific framework paths")
             print("  -h, --help                     print this help");
             print("  -m, --multifiles               launch objj on several files")
+            print("  --xml-output-format            specify the output format in xml.")
```

Comment:
can you add -x so all parameters have their short version? also I think the name should simply be --xml